packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -844,7 +844,11 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
                 class="mb-3"
                 data-model="$ctrl.isAttachFloatingIP"
                 data-on-change="$ctrl.onAttachIPChange(modelValue)"
-                data-disabled="!$ctrl.isFloatingIpAvailable || !$ctrl.PciProjectsProjectInstanceService.additionalIpsIsAvailable($ctrl.flavor.type)"
+                data-disabled="!$ctrl.isFloatingIpAvailable || !$ctrl.PciProjectsProjectInstanceService.additionalIpsIsAvailable($ctrl.flavor.type) || $ctrl.model.number > 1"
+                data-uib-tooltip="{{ $ctrl.model.number > 1 ? ('pci_projects_project_instances_add_attch_floating_ip_checkbox_label_tooltip' | translate) : '' }}"
+                data-tooltip-append-to-body="data-translate-values"
```

Comment:
**nitpick:** Since its a boolean, it makes more readable to set it to `true` than a string which evaluates to true.
```suggestion
                data-tooltip-append-to-body="true"
```